bot/cogs/weather.py

```diff
@@ -0,0 +1,105 @@
+"""Commands for the weather module."""
+import discord
```

Comment:
```suggestion
import aiohttp
import discord

from discord.ext import commands

from bot.settings import WEATHER_TOKEN
```
imports should be grouped, spaced, and basic imports (i.e `import aiohttp` should come before `from` imports)